pkgs/servers/microbin/default.nix

```
@@ -21,6 +23,8 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-7GSgyh2aJ2f8pozoh/0Yxzbk8Wg3JYuqSy/34ywAc2s=";
 
+  patches = lib.optionals (!noPatches) [ ./211.patch ./215.patch ];
```

Can we not just `fetchpatch` these?

```suggestion
  patches = [
    # Some short explanation of the changes here
    # https://github.com/szabodanika/microbin/pull/211
    (fetchpatch {
      name = "0001-microbin-pull-211.patch";
      url = "https://github.com/szabodanika/microbin/compare/b8a0c5490d681550d982ad02d67a1aaa0897f503..3b0c025e9b6dc1ca69269541940bdb53032a048a.patch";
      hash = "";
    })

    # Same for second patch here
  ];
```

---

didn't know how to properly fetchpatch whole pr and survive pushes to it :) thanks

---

done, also in the force push made it the same way for the single commit pr, just wanted uniformity here, I described these as PRs so I wanted to do it the PR way instead of commit way :p